### DISCUSSION TITLE
[Auth Desktop] Option to close program when clicking "X" rather than minimize to system tray

### DISCUSSION THREAD

**Discussion by bowuu** [ente-io/ente#1341] [Enhancements]
On Auth Desktop v2.0.50 (Windows) when clicking "X" to exit the application, Auth will minimize to the system tray instead of fully exiting. 

From https://github.com/ente-io/ente/issues/1210#issuecomment-2020141774 it looks like this is intended behavior, however, it seems like a potential security risk to have a 2FA program always running in the background rather than to only open when needed. 

Even if this is intended behavior, there should be an option to disable closing to system tray so that the user has the option of clicking "X" to fully exit the application.

**Comment by bowuu** [comment]
Closing as building from latest source seems to have fixed the issue.

**Reply by gitmeED331** [reply]
what about those that want the opposite? That want "close-to-tray"?

**Reply by bowuu** [reply]
Should appear as an install option for Windows. No clue on other platforms.

**Reply by gitmeED331** [reply]
linux version does not, the settings options are next to non-existent. There isn't even a cli or man pages.

**Reply by caseyredisatwork** [reply]
> Should appear as an install option for Windows. No clue on other platforms.

That wasn't an option when I installed and minimizing just minimizes to the taskbar, it doesn't "close to tray" and I see no setting for it in either the installed version or the latest beta (auth-v3.0.18-beta.2)
